go.mod

```diff
@@ -3,18 +3,8 @@ module github.com/google/pprof
 go 1.19
 
 require (
-	github.com/chromedp/chromedp v0.9.2
 	github.com/chzyer/readline v1.5.1
 	github.com/ianlancetaylor/demangle v0.0.0-20240312041847-bd984b5ce465
 )
 
-require (
```

Comment:
this reduction in transitive deps for downstreams is exactly what I was hoping to see 🎉